react/features/base/label/components/CircularLabel.native.js

```diff
@@ -11,34 +11,104 @@ import styles from './styles';
 
 type Props = AbstractProps & {
 
+    /**
+     * Extra behaviour of the label. See constants.
+     */
+    extraBehaviour: string,
```

Comment:
How come is this a string if the constants are defined as a `Symbol` ?

Reply:
The naming is kinda not very descriptive here. Can you maybe elaborate a bit on what it actually does so we find a better name for it?